samples/nrf_desktop/CMakeLists.txt

```diff
@@ -71,3 +74,27 @@ add_subdirectory(src/events)
 add_subdirectory(src/hw_interface)
 add_subdirectory(src/modules)
 add_subdirectory(src/services)
+
+if (${CONFIG_BOOTLOADER_MCUBOOT})
+  if (NOT EXISTS ${MCUBOOT_DIR})
+    message(FATAL_ERROR "Path to MCUBoot must be specified")
+  endif()
+  set(cmd ${CMAKE_COMMAND} -E env
+      ${MCUBOOT_DIR}/scripts/imgtool.py sign
+      --key ${MCUBOOT_DIR}/root-rsa-2048.pem
+      --header-size ${CONFIG_TEXT_SECTION_OFFSET}
+      --align 8
+      --version ${CMAKE_PROJECT_VERSION}
+      --slot-size 0x69000
```

Comment:
slot-size will probably not be automated now - it is available from dts and parsed to config but is not visible in cmake for some reason

Reply:
I have found an option that is visible to cmake